helm/azure-operator-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: azure-operator-configmap
+  namespace: giantswarm
+data:
+  config.yaml: |
+    server:
+      listen:
+        address: 'http://0.0.0.0:8000'
+    service:
+      azure:
+        template:
+          uri:
+            version: "master"
```

Comment:
This should be removed. azure-operator defaults to the build git SHA. That way we have predictable template versions.

Reply:
done